test/.eslintrc

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This is only required within the integration tests, right?
So I would prefer to move this into `test/integration`.

Reply:
done